code/modules/power/power.dm

```diff
@@ -426,6 +433,6 @@
 		return null
 	for(var/obj/structure/cable/C in src)
 		if(C.machinery_layer & machinery_layer)
-			C.update_appearance()
+			C.update_appearance() // I hate this. it's here because update_icon_state SCANS nearby turfs for objects to connect to. Wastes cpu time
```

Comment:
fwiw i think you should remove this comment because my PR is coming soon and it mostly fixes the problematic part of this

Reply:
did you fix this? the issue is cables not ever knowing who's actually connected to them, and just scanning constantly